src/main/java/com/lalabrand/ecommerce/security/jwt_token/JwtService.java

```diff
@@ -38,19 +44,20 @@ private SecretKey getKey() {
         return Keys.hmacShaKeyFor(decodedKey);
     }
 
-    public String generateToken(String email, String id) {
+    public String generateToken(String email, String userId, Integer passwordVersion) {
         Map<String, Object> claims = new HashMap<>();
-        claims.put("id", id);
+        claims.put("id", userId);
+        claims.put("passwordVersion", passwordVersion);
         return createToken(claims, email);
     }
 
     private String createToken(Map<String, Object> claims, String email) {
```

Comment:
Why do you need both `createToken` and `generateToken` functions?

Reply:
Good case. I will put createToken functionality into generateToken func.